### PR TITLE
Hotfix/signup ui

### PR DESCRIPTION
**Pull request by abdullah-siddiqui-git**
Following changes are mode in this PR:
-> UI of Signup, login and user related pages are improved.

### CODE REVIEW

**Comment by abdullah-siddiqui-git** [comment]
Cycle - 0 review done.

File: app/assets/stylesheets/signup.css.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+#subdomain_verification_icon {
+  z-index: 999;
+  position: absolute;
+  margin-top: 10px;
+  margin-left: -25px;
+}
+
+
```
Comment (abubakarazeem) [inline app/assets/stylesheets/signup.css.scss]: extra line

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,51 +1,64 @@
-<div class="card">
+<div class="card shodowed_card">
```
Comment (abubakarazeem) [inline app/views/devise/registrations/new.html.erb]: shodowed-card

**Comment by abubakarazeem** [comment]
review cycle 1

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,5 +1,11 @@
 module ApplicationHelper
   def main_layout_class
-    user_signed_in? ? 'col-lg-12' : 'offset-lg-3 col-lg-6 offset-md-2 col-md-8 col-sm-12'
+    user_signed_in? ? 'col-lg-12' : 'offset-lg-1 col-lg-10 offset-md-1 col-md-10 col-sm-12'
+  end
+
+  def main_body_class_provider
+    if !user_signed_in? && request.path != '/'
```
Comment (Anmol79) [inline app/helpers/application_helper.rb]: check for specific actions

**Pull request merged**